src/pages/home/report/ReportActionsList.js

```diff
@@ -335,6 +335,12 @@ function ReportActionsList({
         (reportAction, index) => {
             let shouldDisplay = false;
             if (!currentUnreadMarker) {
+                // Check if the report type is "REPORTREVIEW" and last actor is the current user.
+                // Return shouldDisplay new marker action (terminate flow).
+                // This is to avoid displaying the new line marker when a current userrequests money.
+                if (reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.REPORTPREVIEW && reportAction.childLastActorAccountID === Report.getCurrentUserAccountID()) {
```

Comment:
for checking if it's a report preview you can use isReportPreviewAction from ReportActionsUtils